OpenSim/Common/TRCFileAdapter.cpp

```diff
@@ -133,6 +154,9 @@ TRCFileAdapter::extendRead(const std::string& fileName) const {
     // X1, Y1, Z1, X2, Y2, Z2, ... so on.
     // Check and ignore these labels.
     auto xyz_labels_found = nextLine();
+    // erase blank labels, e.g. due to Frame# and Time columns
+    eraseEmptyElements(xyz_labels_found);
```

Comment:
Just so I understand: the reason the calls to `eraseEmptyElemnts()` is necessary now is because space is no longer a delimiter?